start_cerberus.py

```diff
@@ -315,23 +315,28 @@ def main(cfg):
                             my_check = ".".join(check.replace("/", ".").split(".")[:-1])
                             my_check_module = importlib.import_module(my_check)
                             custom_checks_imports.append(my_check_module)
-
+                    custom_checks_fail_messages = []
+                    custom_checks_status = True
                     for check in custom_checks_imports:
-                        check_returns = check.main()
+                        check_returns, custom_check_message = check.main(iteration)
```

Comment:
Custom checks is designed such that it’s optional to return True/False signal from the custom functions. With these changes, it always becomes necessary to return  a message with go/no-go signal. I think it would be good to support all the 3 variants:
1. no return
2. returns only true/false
3. return true/false and a message

Maybe we can have something like this:
```
return = functionCall()
if type(return) == bool:
    …..
elif type(return) == dict:
   status = return[status]
   message = return[message]
```
Also, I think it would be good to return a dict (for example, `return {“status”: False, “message”: “message…“}`) from custom function instead of a tuple as it handles the case when status and message is returned in different order and the keys in dict would make the function return more self-explanatory. 
Thoughts?

Also please update this [section](https://github.com/cloud-bulldozer/cerberus#bring-your-own-checks) of the README to let the users know that custom failures are now reported on slack channel and the function return format to be followed.

Please run `tox` on the terminal and resolve warnings if any. 
Thanks.